gammapy/spectrum/spectrum_analysis.py

```diff
@@ -1,12 +1,18 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
 import logging
+from astropy import wcs
+
 import numpy as np
 from astropy.coordinates import Angle, SkyCoord
 from astropy.extern import six
-from ..image import ExclusionMask
+from astropy.wcs.utils import skycoord_to_pixel
+
+from gammapy.extern import pathlib
```

Comment:
> @cdeil : Do you know how to teach PyCharm to make import relative when optimizing imports?

I don't think that's possible. It probably would have to be added as an option under Editor -> General -> Auto Import -> Python -> Preferred import style.
The only feature request I could find is this, which at a very quick glance looks different:
https://youtrack.jetbrains.com/issue/PY-9987

I'd also like to have that option ... can you take the time to file a feature request in the PyCharm tracker? Or should I?